src/unix/linux-core.c

```diff
@@ -1011,3 +1011,50 @@ uint64_t uv_get_total_memory(void) {
 
   return 0;
 }
+
+
+static uint64_t uv__read_cgroups_uint64(const char* cgroup, const char* param) {
+  uint64_t rc;
+  ssize_t n;
+  int fd;
+  char filename[256];
+  char buf[32];  /* Large enough to hold an encoded uint64_t. */
+
+  sprintf(filename, "/sys/fs/cgroup/%s/%s", cgroup, param);
+
+  rc = 0;
+  fd = uv__open_cloexec(filename, O_RDONLY);
+
+  if (fd == -1)
+    return 0;
+
+  n = read(fd, buf, sizeof(buf) - 1);
+
```

Comment:
Would it work to close `fd` here? If so, `out:` can be removed.

Reply:
Yep, I've removed the `out:` label.